docs/api/enums.rst

```diff
@@ -2369,6 +2372,14 @@ of :class:`enum.Enum`.
 
     .. versionadded:: 2.5
 
+    .. attribute:: durable
+
+        Represents a durable one-time purchase.
```

Comment:
Straight from the [API docs](https://discord.com/developers/docs/monetization/one-time-purchases#types-of-onetime-purchases)
```suggestion
       A one-time purchase that is permanent and is not subject to either renewal
       or consumption, such as lifetime access to an app's premium features.
```